src/sentry/search/snuba/executors.py

```diff
@@ -217,6 +222,18 @@ def has_sort_strategy(self, sort_by):
         return sort_by in self.sort_strategies.keys()
 
 
+def trend_aggregation(start, end):
+    middle = start + timedelta(seconds=(end - start).total_seconds() * 0.5)
+    middle = datetime.strftime(middle, DateArg.date_format)
+
+    agg_range_1 = "countIf(lessOrEquals(toDateTime('{}'), timestamp))".format(middle)
+    agg_range_2 = "countIf(greater(toDateTime('{}'), timestamp))".format(middle)
```

Comment:
Just want to confirm this is expected behaviour cause its different than how trends is doing this.
```python
agg_range_1 = countIf(middle <= timestamp) # Second half of the time period
agg_range_2 = countIf(middle > timestamp) # First half of the time period
```
So a high result means that the first half's count was larger than the second half (which means an error is showing up less)

Reply:
Dang, got this backwards. I'll add a test to catch.

Reply:
Ahhh okay, these conditions are backwards in `sentry.api.event_search` because of this: https://github.com/getsentry/sentry/blob/master/src/sentry/api/event_search.py#L2196-L2213

Reply:
Ahh, so we literally use the functions backwards when using the new syntax?